diff --git a/bacula-5.0.0/src/lib/message.c b/bacula-5.0.0/src/lib/message.c
index 5791f6b..eed98df 100644
--- a/bacula-5.0.0/src/lib/message.c
+++ b/bacula-5.0.0/src/lib/message.c
@@ -51,7 +51,7 @@ bool dbg_timestamp = false;           /* print timestamp in debug output */
 bool prt_kaboom = false;              /* Print kaboom output */
 utime_t daemon_start_time = 0;        /* Daemon start time */
 const char *version = VERSION " (" BDATE ")";
-char my_name[30];                     /* daemon name is stored here */
+char my_name[64];                     /* daemon name is stored here */
 char host_name[50];                   /* host machine name */
 char *exepath = (char *)NULL;
 char *exename = (char *)NULL;
