commit 71e1f5643d19120fbc71dddf69440897898d34e9
Author: Eric Bollengier <eric@eb.homelinux.org>
Date:   Sat Jan 15 15:21:35 2011 +0100

    Fix JobBytes counter when using Accurate/BaseJobs with checksum

diff --git a/bacula/src/filed/verify.c b/bacula/src/filed/verify.c
index 4750e44..287f0a4 100644
--- a/bacula/src/filed/verify.c
+++ b/bacula/src/filed/verify.c
@@ -370,11 +370,13 @@ static int read_digest(BFILE *bfd, DIGEST *digest, JCR *jcr)
       
       crypto_digest_update(digest, (uint8_t *)buf, n);
 
-      /* Can be used by BaseJobs, update only for Verify jobs */
-      if (jcr->getJobLevel() != L_FULL) {
+      /* Can be used by BaseJobs or with accurate, update only for Verify
+       * jobs 
+       */
+      if (jcr->getJobType() == JT_VERIFY) {
          jcr->JobBytes += n;
-         jcr->ReadBytes += n;
       }
+      jcr->ReadBytes += n;
    }
    if (n < 0) {
       berrno be;
